scout/supporting_data/config_schema.yml

```diff
@@ -210,6 +210,59 @@ properties:
         type: boolean
         default: false
         description: If true, suppress reporting of ECM-captured efficient energy use.
+      sub_fed_policy:
```

Comment:
Can any of the arguments under `sub_fed_policy` possibly conflict with each other or otherwise dependent on each other? Or are they all independently set?